src/index.js

```diff
@@ -1,4 +1,4 @@
-import _ from 'lodash';
+// import _ from 'lodash';
 import './style.css';
 
 // function component() {
```

Comment:
- [ ] <b>Optional:</b> To make sure that your codebase is always clean, remember to remove commented-codes. Kindly, make sure to fix it in the next milestone.